lib/actions/preload.coffee

```diff
@@ -163,8 +169,7 @@ module.exports =
 		.then (docker) ->
 
 			# Build the preloader image
-			buildOutputStream = preload.build(docker)
-			buildOutputStream.pipe(process.stdout)
```

Comment:
>  @pimterry what values are true for process.env.DEBUG ? 1, true, True and TRUE or anything as long as it is defined ?

Yep, it's very simple & dumb: debug output should show if it's set to anything at all.